src/test/java/com/gmail/luchyk/viktoriia/glovodb/service/AddressServiceTest.java

```diff
@@ -0,0 +1,103 @@
+package com.gmail.luchyk.viktoriia.glovodb.service;
+
+import com.gmail.luchyk.viktoriia.glovodb.converter.AddressConverter;
+import com.gmail.luchyk.viktoriia.glovodb.dto.AddressDto;
+import com.gmail.luchyk.viktoriia.glovodb.entity.AddressEntity;
+import com.gmail.luchyk.viktoriia.glovodb.enums.Message;
+import com.gmail.luchyk.viktoriia.glovodb.exception.ObjectNotFoundException;
+import com.gmail.luchyk.viktoriia.glovodb.repository.AddressRepository;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class AddressServiceTest {
+    @Mock
+    private AddressRepository addressRepository;
+
+    private AddressService addressService;
+    private AddressDto expected;
+    private AddressDto updated;
+
+    @BeforeEach
+    public void init() {
+        addressService = new AddressService(addressRepository);
+        expected = AddressDto.builder().id(1).number("1000").street("Drayton str").apartmentNumber(7).build();
+        updated = AddressDto.builder().id(1).number("2000").street("Seven Hills str").apartmentNumber(1).build();
+    }
+
+    @Test
+    public void getTest() throws ObjectNotFoundException {
```

Comment:
please follow given_when_then naming convention. Given state may be omitted for some cases:
getTest() -> whenGet_thenReturnAddress()
also negative cases:
givenNoAddress_whenGet_thenThrowException()